sdks/go/opspec/interpreter/call/container/ports/interpret.go

```diff
@@ -0,0 +1,29 @@
+package ports
+
+import (
+	"strings"
+
+	"github.com/opctl/opctl/sdks/go/model"
+	"github.com/opctl/opctl/sdks/go/opspec/interpreter/reference"
+)
+
+// Interpret container ports
+func Interpret(
+	scope map[string]*model.Value,
+	containerCallSpecPorts map[string]string,
+	scratchDirPath string,
+) (map[string]string, error) {
+	containerCallPorts := map[string]string{}
+	for callSpecContainerPortNumber, callSpecContainerPortExpression := range containerCallSpecPorts {
+		// @TODO: use reference.interpret once reference syntax no longer optional
```

Comment:
Can you avoid this TODO given this is a new type, so it doesn't have backwards compatibility?